app/app.go

```diff
@@ -897,57 +921,6 @@ func initParamsKeeper(appCodec codec.BinaryCodec, legacyAmino *codec.LegacyAmino
 	return paramsKeeper
 }
 
-func (app *App) InitializeAppVersion(ctx sdk.Context) {
```

Comment:
This seems redundant in 0.52 since AppVersion in BaseApp passes directly through to the consensus keeper.